Exercise 3 Recursive Unhider/recursiveUnhider-rt/src/com/tridiumuniversity/recursiveUnhider/BJob.java

```diff
@@ -0,0 +1,47 @@
+package com.tridiumuniversity.recursiveUnhider;
+
+import javax.baja.job.BSimpleJob;
+import javax.baja.naming.BOrd;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+
+@NiagaraType
+public class BJob extends BSimpleJob {
```

Comment:
The Niagara framework happens to already have a class named `BJob` (`javax.baja.job.BJob`), so it'd probably be a good idea to pick a different name for this class.